src/App.css

```diff
@@ -6,5 +6,5 @@
 body {
   margin: 0;
   padding: 0;
-  background-color: black;
```

Comment:
why did you change the color?

Reply:
Its as per the new UI/UX design changes.